bootstrap/src/main/kotlin/jp/co/soramitsu/notary/bootstrap/controller/IrohaController.kt

```diff
@@ -0,0 +1,89 @@
+package jp.co.soramitsu.notary.bootstrap.controller
+
+import jp.co.soramitsu.crypto.ed25519.Ed25519Sha3
+import jp.co.soramitsu.notary.bootstrap.dto.*
+import jp.co.soramitsu.notary.bootstrap.genesis.GenesisInterface
+import mu.KLogging
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import javax.xml.bind.DatatypeConverter
+
+@RestController
+@RequestMapping("/iroha")
+class IrohaController(val genesisFactories: List<GenesisInterface>) {
+
+    private val log = KLogging().logger
+
+    @GetMapping("/config/accounts/{project}/{env}")
+    fun getNeededAccounts(@PathVariable("project") project: String, @PathVariable("env") env: String): ResponseEntity<List<AccountPrototype>> {
+
+        var genesisInfo: GenesisInterface? = null
+        for (gf in genesisFactories) {
+            if (gf.getProject().contentEquals(project) && gf.getEnvironment().contentEquals(env)) {
+                genesisInfo = gf
+            }
+        }
+        val accounts = ArrayList<AccountPrototype>()
+        if (genesisInfo != null) {
+            genesisInfo.getAccountsNeeded().forEach {
```

Comment:
I think we can make the same thing in one line using `filter`

Reply:
fixed